### ISSUE TITLE
Translation of checkbox captions

### ISSUE THREAD

**Issue by 2bad2furious** [czubehead/bootstrap-4-forms#11]
I am experiencing a bug with rendering of checkboxes, the culprit is in the method getControl, where the parent method is called but not used and the new method doesnt translate the label.

I will hopefully write a PR.

**Comment by 2bad2furious** [comment]
Quickfix of CheckboxInput

```
/**
	 * Generates a checkbox
	 * @return Html
	 */
	public function getControl()
	{
		$label = parent::getControl();

		return self::makeCheckbox($this->getHtmlName(), $this->getHtmlId(), $label->getText(), $this->value, FALSE, $this->required, $this->disabled);
	}
```

**Comment by 2bad2furious** [comment]
I was wondering if it's neccessary to call the parent method and not use the result, therefore doing some of the work twice - such as translating, which without the knowledge of its cost could be an expensive operation.

**Comment by 2bad2furious** [comment]
Also, is it neccessary to ignore all and any changes the user might have made to the label or control prototype, such as classes?

**Comment by czubehead** [comment]
I don't know, I haven't dug that deep. I think it was necessary to call the parent method, but I might be wrong. Truth be told, I didn't really focus much on checkboxes.

> Also, is it neccessary to ignore all and any changes the user might have made to the label or control prototype, such as classes?

You can create a pull request solving this issue which I will gladly accept, you clearly have some feature-rich environment which will let you test all that.

**Comment by 2bad2furious** [comment]
I'll try to do that, your library is great so I would love to take the opporturnity to contribute.
Honestly, I just think It would have been easier for you had you just added the classes to the elements.

**** 
Checkbox doesn't really proves the point as it is an example of an input, where the behaviour/rendering is a little bit more complicated. Button, on the other hand, is a perfect example of an element, that only needed a `.btn` class.

**Comment by czubehead** [comment]
Yes, it's certainly slightly over-engineered, but it also allows for great expansion and customisation which would never be possible with KISS approach. This library is designed to be mostly an extensible base which users can relatively easily adjust as they see fit.
